app/src/main/res/layout/fragment_barometer.xml

```diff
@@ -0,0 +1,269 @@
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:layout_marginBottom="@dimen/card_margin"
+                android:layout_marginLeft="@dimen/card_margin"
+                android:layout_marginRight="@dimen/card_margin"
+                android:layout_marginTop="@dimen/card_margin"
+                app:sv_speedTextColor="@color/black"
+                app:sv_speedTextPosition="BOTTOM_CENTER"
+                app:sv_speedometerMode="NORMAL"
+                app:sv_unit="mmHg"
+                app:sv_textSize="@dimen/text_size"
+                app:sv_unitTextSize="@dimen/text_size"
+                app:sv_speedometerBackColor="@color/colorPrimaryDark"
+                app:sv_indicator="NeedleIndicator"
+                app:sv_indicatorColor="@color/black"
+                app:sv_unitTextColor="@color/black" />
+
+        </android.support.v7.widget.CardView>
+
+        <android.support.v7.widget.CardView
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/lux_chart_height"
+            android:layout_marginEnd="@dimen/card_margin"
+            android:layout_marginLeft="@dimen/card_margin"
+            android:layout_marginRight="@dimen/card_margin"
+            android:layout_marginStart="@dimen/card_margin"
+            android:layout_marginTop="@dimen/lux_card_margin_top">
+
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <RelativeLayout
+                    android:id="@+id/chart_xaxis_layout"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_alignParentLeft="true"
+                    android:layout_alignParentStart="true"
+                    android:layout_alignParentTop="true"
+                    android:background="@color/black"
+                    android:orientation="vertical">
+
+                    <TextView
+                        android:id="@+id/tv_graph_label_xaxis_hmc"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_alignParentTop="true"
+                        android:layout_centerHorizontal="true"
+                        android:layout_gravity="center_vertical|center_horizontal|center"
+                        android:foregroundGravity="center_vertical"
+                        android:gravity="center_vertical|center_horizontal|center"
+                        android:text="@string/time"
+                        android:textColor="@color/white"
+                        android:textSize="@dimen/lux_chart_axis_text"
+                        android:textStyle="normal|bold"
+                        tools:layout_editor_absoluteX="@dimen/graph_editor_absoluteX"
```

Comment:
@yatri1609 why are you using absolute X and Y this generally tend to give error in the build and  also the UI doesn't remains stable, please remove it.